guake/__init__.py

```diff
@@ -39,11 +39,7 @@ def vte_version():
 
     from gi.repository import Vte
 
-    s = "{}.{}.{}".format(
-        Vte.MAJOR_VERSION,
-        Vte.MINOR_VERSION,
-        Vte.MICRO_VERSION,
-    )
+    s = f"{Vte.MAJOR_VERSION}.{Vte.MINOR_VERSION}.{Vte.MICRO_VERSION}"
```

Comment:
Not sure why this changed, as this is update at #1880, but in this PR didn't touch black

Reply:
The first commit with the failing build contains all the messages explaining why the build failed, the update to flakehell namely brought with it a strong preference for f strings

Reply:
Great to have them to be f-string!